packages/shared/src/hooks/useRegistration.ts

```diff
@@ -92,6 +94,7 @@ const useRegistration = ({
       method: values.method || 'password',
       csrf_token: getNodeValue('csrf_token', nodes),
       'traits.image': values['traits.image'],
+      'traits.userId': anonymous.id,
```

Comment:
You can also use trackingId directly I think. *(better failsafe)

Reply:
Good point. Let me refactor it quickly.